app/assets/javascripts/live_search.js

```diff
@@ -311,14 +311,18 @@
     this.$loadingBlock.text('Error. Please try modifying your search and trying again.')
   }
 
+  LiveSearch.prototype.updateElement = function updateElement (element, content) {
+    element.html(content)
```

Comment:
This opens us up to potential XSS, as `.html` does not "remove or escape any user input before adding content to the document" ([source](http://api.jquery.com/html/)). So we need to have a look at what we're returning via the JSON request.

Reply:
@bilbof - Isn't the escaping happening on the ERB template?

Reply:
By default it does, but we're using `html_safe` some things which I think will disable this.

Reply:
We're now only using `html_safe` on the next and prev links, which doesn't involve any user input so I think it's safe (plus a few other instances of things that weren't added by this PR but also look safe).